<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="image_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="sample_radioButton">
        <property name="text">
         <string>Sample</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ob_radioButton">
        <property name="text">
         <string>Open Beam</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="di_radioButton">
        <property name="text">
         <string>Dark Current</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="slider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>File name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="filename_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>Dialog</receiver>
   <slot>slider_moved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>640</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider</sender>
   <signal>sliderPressed()</signal>
   <receiver>Dialog</receiver>
   <slot>slider_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>622</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>640</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sample_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>radiobuttons_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ob_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>radiobuttons_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>di_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>radiobuttons_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>slider_value_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>664</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slider_moved()</slot>
  <slot>slider_pressed()</slot>
  <slot>radiobuttons_changed()</slot>
  <slot>slider_value_changed()</slot>
 </slots>
</ui>
